### DISCUSSION TITLE
Integrate with Credential Management API

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5781] [Feature Requests]
See https://developer.mozilla.org/en-US/docs/Web/API/Credential_Management_API

Could create better sign in UX

**Comment by maxpatiiuk** [comment]
Also look into Web Authentication API - https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API

**Comment by grantfitzsimmons** [comment]
This has been requested by CSIRO via [Asana](https://app.asana.com/0/1201972579963460/1207850210383533/f):
>**Improve API authentication**
> Over the past couple of months, a number of side projects required connectivity to our Specify instances utilising the API. Various software approaches have been used and in every instance, people struggled with the authentication part.
> 
> We would like to request a more streamlined approach to API authentication, for example, via an API key/token that can be generated within the Accounts system.
